cmd/main.go

```diff
@@ -72,6 +73,13 @@ func main() { //nolint:funlen // main function needs to initialize many object
 	opts.BindFlags(flag.CommandLine)
 	flag.Parse()
 
+	// get configs from ENV
+	envConfigs, err := env.GetConfig()
+	if err != nil {
+		setupLog.Error(err, "unable to get configs from env")
+		os.Exit(1)
+	}
+
 	// @TODO: Re-check logger setup and init
```

Comment:
Is this todo still relevant?

Reply:
yes, I added a todo here https://github.com/kyma-project/nats-manager/issues/8